parsing.js

```diff
@@ -0,0 +1,6 @@
+const inputElement = document.getElementById("input");
```

Comment:
I think this was a whoopsie miss during a copy-pasta:
```suggestion
const inputElement = document.getElementById("myFile");
```

Reply:
Though see my comment below since you'd be replacing this anyway.